Wrappers.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include "RadioReceiver.h"
+#include "RadioTransmitter.h"
+#include "PowerSupply.h"
+#include "Transceiver.h"
+#include "AmatureRadioStation.h"
```

Comment:
Don't include what you don't use.
None of these types are **used** in this header file.
You aren't accessing any member variables or member functions of these types **in this header file**.
You can forward declare all of these types, instead.
- [ ] revise